src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx

```diff
@@ -133,6 +139,18 @@ export const JobDetailAbout = () => {
             </Button>
           </AuthGuard>
         )}
+        {!isUser && (
+          <AuthGuard>
+            <Button
+              variant="contained"
+              color="error"
+              customStyle="hidden md:block w-full"
+              onClick={() => navigate(`/nowruz/jobs/list`)}
+            >
+              Close
```

Comment:
if user is not logged-in, user is considered as 'user' type, not org, so the <AuthGuard> is redundant here, because if user is not logged-in, the close button should not be displyed